### DISCUSSION TITLE
Does oracledb thin connect usage with OCI transactional DB

### DISCUSSION THREAD

**Discussion by ptekelly** [oracle/python-oracledb#153] [Q&A] [answered]
Hi 
I am using the always free tier of OCI and have a transactional database that will stop after 7 days of non usage (and eventually be deleted after 30 or so)

in the email i get from OCI i get this
_**"If you plan to actively use your free database in the near future, please take a moment today to make a connection to it over HTTPS or Oracle Net Services (SQL*Net) or use the link below to connect directly to Database Actions landing pad:"**_

What I wanted to know is - if I use oracledb in python to connect to my database using the thin connection will this satisfy the above conditions to keep my database up and running?

My connection and call is something like this (i've removed loads of stuff that prob of no interest)
_**pool = oracledb.create_pool(user=user, password=pwd, dsn=dsn,min=2, max=5, increment=1)
connection = pool.acquire()
cur = connection.cursor()
cur.callproc("DISCO_LOG_MESSAGE",[log_type,message,timestamp,db_log_status,db_log_message])
pool.close()**_

**Comment by cjbj** [comment, marked as answer]
Connecting via Thin mode certainly stops my Always Free DBs from being shutdown.

**Reply by ptekelly** [reply]
superb - thanks for clarification - As I am not consistently developing I didn't want to assume it was just my lack of work that was causing it!  
just need the async now and am good to go
